model/src/main/kotlin/util/FindingsMatcher.kt

```diff
@@ -166,6 +166,6 @@ class FindingsMatcher(
                 locationsForLicenses[license] ?: sortedSetOf(),
                 copyrightsForLicenses[license] ?: sortedSetOf()
```

Comment:
`locationsForLicenses` and `copyrightsForLicenses` also do not need to be sorted, right?

Reply:
done.